src/test/resources/org/jenkinsci/plugins/casc/core/UnsecuredAuthorizationStrategyConfiguratorTest.yml

```diff
@@ -0,0 +1,8 @@
+jenkins:
+  securityRealm:
+    local:
+      allowsSignup: false
+      users:
+        - id: "admin"
+          password: "1234"
```

Comment:
please avoid clear text password in code sample and test 
(see https://github.com/jenkinsci/configuration-as-code-plugin/commit/7ad13d861867b9f490df25b249adced940516748)